### ISSUE TITLE
Add role management layer in deep-security

### ISSUE THREAD

**Issue by CCristi** [MitocGroup/deep-framework#499]
Role creation, listing, removal should pass through deep-security module

**Comment by ivignan** [comment]
@CCristi Is this still relevant?

**Comment by CCristi** [comment]
Yes, currently role management is done by deep-account microservice. The role management layer should be migrated to deep-framework
